### PR TITLE
Add all parameters of the scrollable

### PR DESCRIPTION
**Pull request by cp-sneha-s**
- Support all the parameters of the scrollable

### CODE REVIEW

File: example/lib/home_page.dart [hunk 1]
```diff
@@ -21,11 +21,12 @@ class HomePage extends StatefulWidget {
 class _HomePageState extends State<HomePage> {
   AnimationType appliedStyle = AnimationType.fadeIn;
   List<User> list =
-      List.generate(3, (index) => User(name: "User $index", index: index));
+      List.generate(50, (index) => User(name: "User $index", index: index));
```
Comment (cp-radhika-s) [inline example/lib/home_page.dart:24]: Revert unnecessary changes from example

**Review by cp-radhika-s — CHANGES_REQUESTED** [review]

File: example/lib/home_page.dart (hunk 1 above)
Comment (cp-sneha-s) [reply example/lib/home_page.dart:24]: done

Comment (cp-sneha-s) [reply example/lib/home_page.dart:24]: done
